src/server/routers/account.ts

```diff
@@ -1,16 +1,24 @@
 import { isAuthed } from '@/server/middleware';
 import { t } from '@/server/trpc';
+import { z } from 'zod';
 
 export const accountRouter = t.router({
-  me: t.procedure.use(isAuthed).query(async ({ ctx }) => {
-    const accounts = await ctx.db.account.findMany({
+  me: t.procedure.use(isAuthed)
+  .input(z.object({
+    provider: z.enum([
+      'github',
+      'gitlab'
+    ]),
+  }))
+  .query(async ({ ctx, input }) => {
+    const account = await ctx.db.account.findFirst({
```

Comment:
I think that Prettier is not setup correctly on you IDE. Please check that you have Prettier installed and that `formatOnSave` in your IDE is set to `true`.